phasmophobia_evidence_v3/widget.js

```diff
@@ -68,6 +79,8 @@ window.addEventListener('onWidgetLoad', function (obj) {
   toggleOptObjThree = fieldData['toggleOptObjThree'];
   vipToggleOnCommand = fieldData['vipToggleOnCommand'];
   vipToggleOffCommand = fieldData['vipToggleOffCommand'];
+  mapNameCommand = fieldData['mapNameCommand'];
+  mapDiffCommand = fieldData['mapDiffCommand'];
```

Comment:
Could possibly add an option for the Map Name command to also capture Map Difficulty if it is provided (Similar to how `!ghostreset` can also set the name of the ghost.

Reply:
This is actually possible to do using the map command already, so you can set just map or map and difficulty with the map command.